File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        l_s = 0
        nums_hash = Counter(nums)
        starters = []

        for num in nums_hash:
            if num - 1 not in nums_hash:
                starters.append(num)

        for starter in starters:
            count = 1
            seq_end = starter + 1

            while seq_end in nums_hash:
                count += 1
                seq_end += 1

            if count > l_s:
                l_s = count

        return l_s

File: Data Structures & Algorithms/sort-an-array/submission-0.cpp
class Solution {
public:
    vector<int> sortArray(vector<int>& nums) {
        int i = 0;
        int cnt = 0;

        while (i < nums.size() - cnt - 1) {
            int j = i;
            if (nums[i] > nums[i+1]) {
                while (j < nums.size() - cnt - 1 && nums[j] > nums[j+1]) {
                        int dummy = nums[j+1];
                        nums[j+1] = nums[j];
                        nums[j] = dummy;
                        j++;
                }
                i = 0;
            } else {
                i++;
            }
        }
        return nums;
    }
};